geotrellis-sentinelhub/src/test/scala/org/openeo/geotrellissentinelhub/PyramidFactoryTest.scala

```diff
@@ -1028,4 +1038,40 @@ class PyramidFactoryTest {
     verify(catalogApiSpy, atLeastOnce()).search(eqTo("sentinel-1-grd"), any(), eqTo(LatLng),
       eqTo(from), eqTo(ZonedDateTime.parse("2020-04-09T23:59:59.999999999Z")), any(), eqTo(util.Collections.emptyMap()))
   }
+
+  @Test
+  def testIssue128(): Unit = {
+    val endpoint = "https://services.sentinel-hub.com"
+
+    val catalogApiSpy = spy(new DefaultCatalogApi(endpoint))
```

Comment:
Can be a regular `DefaultCatalogApi` instead of a spy.

Reply:
Changed